### PR TITLE
Blog second section

### PR DESCRIPTION
**Pull request by wevekamola**
PR for #67 

- Single Page UI
- Social media Share
- Comment Box
- Recommend Reading

### CODE REVIEW

File: public/wp-content/themes/glific/dist/images/blog-delaut-image.svg [hunk 1]
```diff
@@ -1 +1,2 @@
...
...LNTMcR24Fkwg6ARImJX+clRXuWjd9gga6ZqpKxODE7PpIYOBcnZMwAoEHBnZULJdvyENSZYn5azdZSpPRizUrHAHch2ZgIEESn4+Ao49J3TnSA5VCtdmLtCdkcEZsFAzGChnxwTMSkAucUDJ5oN69nX+1TQJiiLCwyYU/XXENL9YqJnmUnBFmEDgCJBSbfHG/SAXlBpUiDTx3Le7Mw3KzNBsWKgZipMzYwLmIkC9s9IdRxVXkCK+Bjy1BhcHPkVPUW95OmeGYyzUzHAVuA5MwGgC6DCo9LdccOw8Jm7s0XOdMmPjbfeacdjpri4LNTcJ/mYCoUAAdwU49udCKS4EyEVGDTXdYKRzYWG2G8ykk+auWdlvFmplafBvJmBFAmhUw0l+ag+cBsdhNBuEbhsND5JULNltw8+vythheN4GZ8hCzWCgnB0TCDiBP33ROrPR6XbOOXS8nQ9yifi+TRX1daEH0FsLV2VUqUMVFfVUoogLNbS5RPouHJgAEwggAfTqRI6zZQc6QUHBJaMzbVdBSQALrJS1jIbSxhemZ35e6YxJD4gLNXKhZYACn0m5cLWYABOQJAeudI4tWrN3rpVgiAs1K7WS68oEmIBWAgV2SRpZsGbv11oTVnV8FmpVfQW4fCZgMgK4/SlHkuRrUKBtMlnVVFWHrXSowsSRmED1IIAC7TvJLnUuTM+ypECjq8RCrXrcq9xKJuCPgIzGHmcMq9epb+HqvZZeAeThp79LzeeZQIgTwN7Zb3IYjCtalfntAsiyfGu5p2b5S8gNYAKiBKQSXN18NrFeYmoxCjTRXMyWjntqZrsiXB8mEAQC6ChlRbjd/kDeqj17D4MpjW0IU2ChJoyOEzIBCxKQpKU2u/25wlV7Npp5U7oesizU9NDjtEzAEgSkUgnkL1DvbMb59MztlqiyjkqyUNMBj5MyATMTwAWALbIkzY0A+DQvPfOkmetqZN1YqBlJk/NiAlVLoAAF2feoOLs6DGxf5KXv3UPVKa7aOgW9dBZqQUfOBTIBAwhI0nm0ZLsXJ/wzMLddqHG6rmWdiE27FuwK6m53A1pieBZhSTmRpbk26THDc+YMmQAT0E9AhmJZkvMkF+ShPbN8WZbOgew8iFuYjqA6Bprw+Cvs+utntf71/4wQBbx5YvjXAAAAAElFTkSuQmCC"></image></g></g></g></svg>
```
Comment (Satendra-SR) [inline public/wp-content/themes/glific/dist/images/blog-delaut-image.svg]: Just confirming if this is a minify SVG.

File: public/wp-content/themes/glific/functions.php [hunk 2]
```diff
@@ -158,3 +158,44 @@ function show_more_blogs()
...
 }
+
+function wpb_move_comment_field_to_bottom($fields)
+{
+	$comment_field = $fields['comment'];
+	unset($fields['comment']);
+	$fields['comment'] = $comment_field;
+	return $fields;
+}
+
```
Comment (Satendra-SR) [inline public/wp-content/themes/glific/functions.php]: Please remove line break for consistency.

File: public/wp-content/themes/glific/functions.php [hunk 3]
```diff
@@ -158,3 +158,44 @@ function show_more_blogs()
 if (function_exists('acf_add_options_page')) {
 	acf_add_options_page();
 }
+
+function wpb_move_comment_field_to_bottom($fields)
```
Comment (Satendra-SR) [inline public/wp-content/themes/glific/functions.php]: @wevekamola is it a typo or there is some reason we are using `wpb`?

File: public/wp-content/themes/glific/functions.php [hunk 4]
```diff
@@ -158,3 +158,44 @@ function show_more_blogs()
...
+{
+	$arg['url'] = '';
+	return $arg;
+}
+add_filter('comment_form_default_fields', 'wp_remove_comment_url');
+
+function wp_add_comment_fields($fields)
+{
+	$fields['company'] = '<p class="comment-form-company"><label for="company">' . __('Company name *') . '</label>' .
+		'<input id="company" name="company" type="text" size="30" /></p>';
```
Comment (Satendra-SR) [inline public/wp-content/themes/glific/functions.php]: format?

File: public/wp-content/themes/glific/single.php [hunk 5]
```diff
@@ -25,9 +25,96 @@
...
 		<?php $featured_image_url = get_the_post_thumbnail_url() ?: get_template_directory_uri() . '/dist/images/blog-delaut-image.svg'; ?>
 		<div class="w-90p h-200 w-md-741 h-md-451 w-xl-1092 h-xl-641 mx-auto bg-position-center bg-size-cover rounded-30" style="background-image: url(<?php echo $featured_image_url; ?>);"></div>
-		<div class="w-90p w-md-615 mt-15 ml-8 text-black font-heebo-regular fz-18 leading-27 mb-26">
-			<?php echo the_content(); ?>
+		<div class="d-flex w-md-741 w-xl-1092 flex-column flex-md-row-reverse justify-content-between">
+			<div class="mt-9 mt-md-15 w-full w-md-50 d-flex flex-column align-items-center">
+				<p class="font-heebo-regular fz-18 leading-27 mb-5">Share</p>
+				<div class="d-flex justify-content-between align-items-center flex-row flex-md-column">
+					<div class="w-40 h-40 mb-7 rounded-circle bg-theme-mine-shaft">
+						<a target="_blank" href="https://www.facebook.com/sharer.php?u=<?php echo esc_url(get_permalink()); ?>&t=<?php echo get_the_title(); ?>">
```
Comment (Satendra-SR) [inline public/wp-content/themes/glific/single.php:33]: I see the difference in code formatting in this file and above the `page-blogs.php` file. Please check which one we are going to have and update accordingly.

**Review by Satendra-SR — APPROVED** [review]
@wevekamola approving with few minor changes. Please have a look.

File: public/wp-content/themes/glific/single.php (hunk 5 above)
Comment (wevekamola) [reply public/wp-content/themes/glific/single.php:33]: @Satendra-SR The updated are the formatted one. 👍

File: public/wp-content/themes/glific/functions.php (hunk 4 above)
Comment (wevekamola) [reply public/wp-content/themes/glific/functions.php]: Done 👍

File: public/wp-content/themes/glific/functions.php (hunk 3 above)
Comment (wevekamola) [reply public/wp-content/themes/glific/functions.php]: @Satendra-SR Yes, it was supposed to be `wp`. Thanks for catching that. I have updated it

File: public/wp-content/themes/glific/functions.php (hunk 2 above)
Comment (wevekamola) [reply public/wp-content/themes/glific/functions.php]: @Satendra-SR done 👍

File: public/wp-content/themes/glific/dist/images/blog-delaut-image.svg (hunk 1 above)
Comment (wevekamola) [reply public/wp-content/themes/glific/dist/images/blog-delaut-image.svg]: @Satendra-SR I have corrected that one extra space. For this one, I had to minify it manually as after minify from the various tools the image was getting corrupted.

**Pull request merged**